rollup/cmd/mock_app.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"scroll-tech/common/cmd"
 	"scroll-tech/common/docker"
```

Comment:
can remove this import, and also the source files.

Reply:
It is still needed now, and some docker has not been completely replaced. I will submit a few more PRs later.